devicon.json

```diff
@@ -6587,7 +6587,7 @@
                 "plain"
             ]
         },
-        "color": "#412846",
+        "color": "#000",
```

Comment:
I recommend we pick a different color than black here.
Since they don't have any official primary color as far as I can see here are some candidates from their website:

`#ba2bd2` from their wordmark and diagram backgrounds
![image](https://github.com/user-attachments/assets/a96edfd3-1850-4af5-895d-0b7a1c6c4b39)

`#a829c3` from their header background
![image](https://github.com/user-attachments/assets/fd71848f-8bea-4105-8fa5-8d11624d39f9)

`#a829c3` from their hovered links or `#dc65ff` when they are not hovered
![image](https://github.com/user-attachments/assets/b267f176-6839-4850-9d32-de6fae9ca27f)

`#9116b2` from their footer
![image](https://github.com/user-attachments/assets/8c3e0c39-3cf3-4e41-98bb-6e0f4867efd3)


Personally I prefer the one they use in the diagram and wordmark, but I'll let you pick the one you think fits best, since they are all very similar :)
```suggestion
        "color": "#ba2bd2",
```